noark-extraction-validator/src/main/resources/noark53/noark53-validation.xml

```diff
@@ -356,58 +370,556 @@
+        </queries>
+    </check>
+
+    <check id="ASC16">
+        <title>Precedent count</title>
+        <description>Number of Precedents found for Files and Records grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT s.systemid series_system_id,
+                        COALESCE(f.val, 0) number_of_file_precedents,
+                        COALESCE(r.val, 0) number_of_record_precedents
+                    FROM arkivstruktur.arkivdel s
+                    LEFT JOIN
+                    (
+                        SELECT f._ref_arkivdel series_system_id, COUNT(*) val
+                        FROM arkivstruktur.presedens p
+                        JOIN arkivstruktur.mappe f ON f.systemid = p._ref_mappe
+                        GROUP BY f._ref_arkivdel
+                    ) f ON f.series_system_id = s.systemid
+                    LEFT JOIN
+                    (
+                        SELECT r._ref_arkivdel series_system_id, COUNT(*) val
+                        FROM arkivstruktur.presedens p
+                        JOIN arkivstruktur.registrering r ON r.systemid = p._ref_registrering
+                        GROUP BY r._ref_arkivdel
+                    ) r ON r.series_system_id = s.systemid;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC17">
+        <title>Sign Off count</title>
+        <description>Number of Sign Offs grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT r._ref_arkivdel series_system_id, COUNT(*) number_of_sign_offs
+                    FROM arkivstruktur.registrering r
+                    WHERE r._dtype = 'journalpost' AND r.systemid in (SELECT _ref_registrering FROM arkivstruktur.avskrivning WHERE _ref_registrering IS NOT NULL GROUP BY _ref_registrering)
+                    GROUP BY r._ref_arkivdel;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC18">
+        <title>Document Flow count</title>
+        <description>Number of Document Flows grouped by Series</description>
+        <group>arkivstruktur</group>
+        <queries>
+            <info>
+                <![CDATA[
+                    SELECT r._ref_arkivdel series_system_id, COUNT(*) number_of_document_flows
+                    FROM arkivstruktur.registrering r
+                    WHERE r._dtype = 'journalpost' AND r.systemid in (SELECT _ref_registrering FROM arkivstruktur.dokumentflyt WHERE _ref_registrering IS NOT NULL GROUP BY _ref_registrering)
+                    GROUP BY r._ref_arkivdel;
+                ]]>
+            </info>
+        </queries>
+    </check>
+
+    <check id="ASC19">
+        <title>Grading count</title>
+        <description>
+            Number of Grading objects related to Series, Classes,
```

Comment:
`Number of gradings associated with series, classes, files, records, or document descriptions`